opm/material/components/SimpleHuDuanH2O.hpp

```diff
@@ -183,14 +183,16 @@ class SimpleHuDuanH2O : public Component<Scalar, SimpleHuDuanH2O<Scalar>>
 
     /*!
      * \brief Specific enthalpy of liquid water \f$\mathrm{[J/kg]}\f$.
+     *        Made by fitting a 2nd-degree polynomial to Coolprop data
+     *        for temperatures in the liquid range at 1 atm.
      *
      * \param temperature temperature of component in \f$\mathrm{[K]}\f$
      * \param pressure pressure of component in \f$\mathrm{[Pa]}\f$
      */
     template <class Evaluation>
     static Evaluation liquidEnthalpy(const Evaluation& temperature,
                                      const Evaluation& /*pressure*/)
-    { return 4180*temperature; }
+    { return 4173.90253918*temperature + 0.11463337*pow(temperature, 2); }
```

Comment:
Is there a reason for using `pow(x,2)` instead of something along the lines of
```C++
return temperature * (4173.90253918 + 0.11463337*temperature);
```
?  I would expect the latter to be less expensive and, possibly, more accurate.

Reply:
Thanks for pointing this out! I have changed it.